pkg/utils/gardener/identity.go

```diff
@@ -135,3 +135,17 @@ type APIServerInfo struct {
 	// WorkloadIdentityIssuerURL is the URL of the issuer for WorkloadIdentities.
 	WorkloadIdentityIssuerURL string `json:"workloadIdentityIssuerURL" yaml:"workloadIdentityIssuerURL"`
 }
+
+// MaintainSeedProviderLabels maintains the seed.gardener.cloud/provider=<type> and seed.gardener.cloud/region=<region> labels on the given object.
+func MaintainSeedProviderLabels(obj client.Object, providerType, providerRegion string) {
```

Comment:
This func is unused. In https://github.com/gardener/gardener/pull/12623#discussion_r2287941830 I already suggested its removal. Can we remove it?